packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Customize the Okta URL domain
+excerpt: Learn how to add a custom domain name to your Okta organization.
+layout: Guides
+---
+
+This guide teaches you how to customize the Okta URL domain, meaning that Okta-hosted login pages will be served with your own branded URL&mdash;an important step towards creating a seamless experience for your users.
```

Comment:
Ha, one more nitpick on my own text — we should remove "login", as it isn't just about login pages — it also encompasses error pages, etc.?